makefile

```diff
@@ -1,6 +1,6 @@
 CC ?= gcc  # version?
 CXX ?= g++
-CPPFLAGS = -std=c++17 -Wall -Wextra -Wconversion -Werror -fno-rtti -fno-exceptions $(EXTRA_CPPFLAGS)
```

Comment:
Omit this from this patch.  This can be landed separately in https://github.com/dsLeks/XOS-Language/pull/41